pkg/crc/network/nameservers.go

```diff
@@ -34,13 +36,48 @@ func GetResolvValuesFromInstance(sshRunner *ssh.Runner) (*ResolvFileValues, erro
 func CreateResolvFileOnInstance(sshRunner *ssh.Runner, resolvFileValues ResolvFileValues) error {
 	resolvFile, _ := CreateResolvFile(resolvFileValues)
```

Comment:
This can be moved to the `replaceResolvConfFile` block, or even to the `replaceResolvConf` helper.

Reply:
Done